### PR TITLE
Don't specify images to preload

### PR DESCRIPTION
**Pull request by mkolesnik**
Shipyard now preloads all built images,
there's no need to explicitly preload any images.

Depends on https://github.com/submariner-io/shipyard/pull/937

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,7 +37,6 @@ dockertogoarch = $(patsubst arm/v7,arm,$(1))
 
 PLATFORMS ?= linux/amd64,linux/arm64
 IMAGES = submariner-operator submariner-operator-index
-PRELOAD_IMAGES := submariner-operator
```
Comment (tpantelis) [inline Makefile:40]: This will now preload the `submariner-operator-index` image as well - is this desired?

Comment (mkolesnik) [reply Makefile:40]: Its benign and would actually be needed when testing any bundle based deployments in the future.

**Review by tpantelis — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Comment by dfarrell07** [comment]
[E2E (1.24)](https://github.com/submariner-io/submariner-operator/actions/runs/3037150478/jobs/4889237887#logs) and [Deployment (globalnet)](https://github.com/submariner-io/submariner-operator/actions/runs/3037150477/jobs/4889238194#logs) are failing, rerunning.

**Comment by dfarrell07** [comment]
[E2E (1.22)](https://github.com/submariner-io/submariner-operator/actions/runs/3039503041/jobs/4894496243#logs) timed out, trying again

**Pull request merged**